.azure-pipelines/benchmarks.yml

```diff
@@ -0,0 +1,75 @@
+trigger:
+  branches:
+    include:
+      - master
+      - refs/tags/*
+pr: none
+
+variables:
+  buildConfiguration: release
+  dotnetCoreSdkVersion: 3.1.107
+  ddApiKey: $(DD_API_KEY)
+  publishOutput: $(Build.SourcesDirectory)/src/bin/managed-publish
+
+# Declare the datadog agent as a resource to be used as a pipeline service
+resources:
+  containers:
+  - container: dd_agent
+    image: datadog/agent:latest
+    ports:
+    - 8126:8126
+    env:
+      DD_API_KEY: $(ddApiKey)
+      DD_HOSTNAME: none
+      DD_APM_ENABLED: true
+      DD_APM_NON_LOCAL_TRAFFIC: true
```

Comment:
I honestly think we should wait for the agent release where we are going to have a dedicate variable for the CI mode, before merging this PR.